### PR TITLE
Fix google analytics and update TA section

### PR DESCRIPTION
**Pull request by elgeish**
Still don't have access to deploy.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -147,22 +148,20 @@ course:
         - name: Jonathan Braatz
           type: On-campus TA
           url: 
-          image: unknown.png
+          image: jon.png
```
Comment (haossr) [inline _config.yml:151]: jpg?

File: _config.yml [hunk 2]
```diff
@@ -147,22 +148,20 @@ course:
...
           url: https://elgeish.com
           image: mohamed.jpg
         - name: Sarah Ciresi
           type: On-campus TA
           url: https://www.linkedin.com/in/sarahciresi
-          image: unknown.png 
+          image: sarah.png
         - name: Vineet Kosaraju
           type: On-campus TA
           url: http://vineetkosaraju.com
```
Comment (haossr) [inline _config.yml:166]: jpg?

File: _config.yml (hunk 1 above)
Comment (elgeish) [reply _config.yml:151]: Fixed

File: _config.yml (hunk 2 above)
Comment (elgeish) [reply _config.yml:166]: Fixed

**Pull request merged**